helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -105,7 +105,7 @@ public BestPossibleExternalViewVerifier(RealmAwareZkClient zkClient, String clus
     _dataProvider = new ResourceControllerDataProvider();
   }
 
-  private BestPossibleExternalViewVerifier(RealmAwareZkClient zkClient, String clusterName,
+  protected BestPossibleExternalViewVerifier(RealmAwareZkClient zkClient, String clusterName,
```

Comment:
This breaks the builder design pattern a little bit.
I'm thinking if we have a better way.

What if we add a configuration to the Verifier Builder called "waitUntilVerify"? It can be a generic configuration for all verifiers even production ones.
1. It is optional, so no backward compatibility issue.
2. It potentially helps our customers too, because they may face the same verify timing issue.
3. It requires the same amount of change as your current design. But no extra class introduced.

Please consider adding it to ZkHelixClusterVerifier.Builder, so all the verifier gets it.

Reply:
refactored.